core/src/components/checkbox/sdds-checkbox.stories.tsx

```diff
@@ -48,19 +48,31 @@ export default {
         defaultValue: { summary: false },
       },
     },
+    indeterminate: {
+      name: 'Indeterminate',
+      description: 'Set the checkbox state to indeterminate',
+      control: {
+        type: 'boolean',
+      },
+      table: {
+        defaultValue: { summary: false },
+      },
+    },
```

Comment:
Really great stuff! I would recommend adding a "if" here to not let the user set the checkbox both checked and indeterminate.

Something like:
if: { arg: 'checked', neq: true },